includes/class-options.php

```diff
@@ -30,6 +28,45 @@ public static function init() {
 		\add_filter( 'option_activitypub_max_image_attachments', array( self::class, 'default_max_image_attachments' ) );
 	}
 
+	/**
+	 * Delete all options.
+	 */
+	public static function delete() {
+		\delete_option( 'activitypub_actor_mode' );
```

Comment:
Now that we touch this, I wonder if we should switch it to a `$wpdb->delete()` call that removes all `activitypub_*` options. That way we wouldn't have to keep updating it.

Reply:
but we would bypass possible hooks!?!

Reply:
But let's give it a try... I will change it to use $wpdb->delete()

Reply:
It would not trigger those hooks, yeah. Is there anything depending on them?

Reply:
not from the core AP plugin

Reply:
If it comes up in the future, I'd recommend other plugins hook into the uninstall hook since that is where this function runs. Good catch.